explorer/src/services/password.ts

```diff
@@ -4,6 +4,15 @@ export function hash(plaintext: string): Promise<string> {
   return argon2.hash(plaintext)
 }
 
-export function compare(plaintext: string, hash: string): Promise<boolean> {
-  return argon2.verify(hash, plaintext)
+export async function compare(
+  plaintext: string,
+  hash: string,
+): Promise<boolean> {
+  try {
```

Comment:
The function body can consist of this instead
```suggestion
  return argon2.verify(hash,plaintext).catch(console.error)
```

Reply:
then we dont need an async function too

Reply:
This doesn't return a boolean